moabb/pipelines/utils_pytorch.py

```diff
@@ -0,0 +1,144 @@
+from collections import Counter
+from functools import partial
+from inspect import getmembers, isclass, isroutine
+
+from braindecode.datasets import BaseConcatDataset, create_from_X_y
+from numpy import unique
+from sklearn.base import BaseEstimator, TransformerMixin
+from skorch.callbacks import Callback
+from torch.nn import Module
+
+
+class Transformer(BaseEstimator, TransformerMixin):
```

Comment:
Better use a more meaningful name, Transformer is too generic.
What about BraindecodeDatasetLoader ? or SkorchDatasetLoader?